src/build.py

```diff
@@ -904,9 +912,11 @@ def OCaml():
 
 def Spec():
   buildbot.Step('spec')
-  # Spec builds in-tree. Always clobber and run the tests.
+  # Spec builds in-tree. Always clobber.
   proc.check_call(['make', 'clean'], cwd=ML_DIR)
-  proc.check_call(['make', 'all'], cwd=ML_DIR)
+  proc.check_call(['make', 'opt'], cwd=ML_DIR)
```

Comment:
Yes opt == wasm == the spec interpreter.

I would put "wasm" here instead.. which might make more sense because that is what the copy command below expects.